app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt

```diff
@@ -208,7 +216,9 @@ fun AddActivityScreen(
         Activity(
             title = title,
             description = description,
-            location = Location(country = "Unknown", city = "Unknown", county = null, zip = null),
+            location =
+                selectedPlace?.let { Location(address = it.place.displayName ?: "") }
```

Comment:
Same here, we should store the place as a Custom place not as a simple location.

Reply:
Same as above.